src/bootloader/bootloader.c

```diff
@@ -310,6 +310,30 @@ static void _render_default_screen(void)
     UG_SendBuffer();
 }
 
+extern bool bootloader_pairing_request;
+extern uint8_t bootloader_pairing_code_bytes[16];
+
+void bootloader_render_ble_confirm_screen(bool confirmed)
+{
+    bootloader_pairing_request = true;
+    uint32_t pairing_code_int = (*(uint32_t*)&bootloader_pairing_code_bytes[0]) % 1000000;
+    unsigned int part0 = pairing_code_int / 10000;
+    unsigned int part1 = (pairing_code_int / 100) % 100;
+    unsigned int part2 = pairing_code_int % 100;
+    char code_str[10] = {0};
+    snprintf(code_str, sizeof(code_str), "%02u %02u %02u", part0, part1, part2);
```

Comment:
Would be nice to show a title here, like "Pairing code" (same as in the noise pairing).

Reply:
done